CFC_WebApp/tests/client_tests/TestSocialMedia.py

```diff
@@ -0,0 +1,20 @@
+import logging
+import unittest
+from dao.socialmediauser import FacebookUser
+from dao.user import User
+from get_database import get_db
+
+logging.basicConfig(level=logging.DEBUG)
+
+class TestSocialMedia(unittest.TestCase):
+    def setUp(self):
+        from tests.common import dropAllCollections
+        dropAllCollections(get_db())
+        user = User.register("oqsaqjk_adeagbosky_1432921786@tfbnw.net")
+        self.uuid = user.uuid
+
```

Comment:
Will do.
Sid Feygin
sid.feygin@gmail.com



> On Jun 2, 2015, at 11:36 PM, shankari <notifications@github.com> wrote:
> 
> In CFC_WebApp/tests/client_tests/TestSocialMedia.py <https://github.com/e-mission/e-mission-server/pull/133#discussion_r31597049>:
> 
> > @@ -0,0 +1,20 @@
> > +import logging
> > +import unittest
> > +from dao.socialmediauser import FacebookUser
> > +from dao.user import User
> > +from get_database import get_db
> > +
> > +logging.basicConfig(level=logging.DEBUG)
> > +
> > +class TestSocialMedia(unittest.TestCase):
> > +    def setUp(self):
> > +        from tests.common import dropAllCollections
> > +        dropAllCollections(get_db())
> > +        user = User.register("oqsaqjk_adeagbosky_1432921786@tfbnw.net")
> > +        self.uuid = user.uuid
> > +
> Shouldn't you call some of the test methods from in here? Otherwise, how do you know that they are invoked?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/e-mission/e-mission-server/pull/133/files#r31597049>.
>